brain/compartmentReportView.cpp

```diff
@@ -163,7 +113,7 @@ std::future<CompartmentReportFrames> CompartmentReportView::load(float start,
         for (size_t i = std::floor(start / timestep); i * timestep < end; ++i)
```

Comment:
The code here is now exactly the same as in load(timestamp) because here it's assumed that the report starts at a time which is multiple of the timestep. I don't know if it could not be the case.